Components/imgComponent/styles.css

```diff
@@ -0,0 +1,25 @@
+@import url(/rootStyles.css);
+
+.img-container {
+  padding: 1rem;
+}
+.img-responsive.img-round {
+  border: 1px solid black;
+  border-radius: 50%;
+}
+
+@media screen and (min-width: 800px) {
```

Comment:
If you have to use media-queries then your img is not responsive, that's what i think.

Reply:
isnt the point of using media queries is to make it responsive, thats why i used this